src/components/CreateWallet.jsx

```diff
@@ -116,10 +53,136 @@ export default function CreateWallet({ currentWallet, startWallet }) {
           )}
         </rb.Button>
       </rb.Form>
-    ) : (
-      <rb.Alert variant="warning">
-        Currently <strong>{walletDisplayName(currentWallet.name)}</strong> is active. You need to lock it first.
-      </rb.Alert>
-    )
+    </>
+  )
+}
+
+const WalletCreationConfirmation = ({ createdWallet, walletConfirmed }) => {
+  const [userConfirmed, setUserConfirmed] = useState(false)
+
+  const onSwitch = (e) => {
+    setUserConfirmed(e.target.checked)
+  }
+
+  return (
+    <div>
+      <p className="mb-4">
+        <div>Wallet Name</div>
+        <div className="fs-4">{walletDisplayName(createdWallet.name)}</div>
+      </p>
+      <p className="mb-4">
+        <div className="mb-2">Seedphrase</div>
+        <Seedphrase seedphrase={createdWallet.seedphrase} />
+      </p>
+      <p className="mb-4">
+        <div>Password</div>
+        <div className="fs-4">{createdWallet.password}</div>
+      </p>
+      <p className="mb-4">
+        <rb.Form.Switch onChange={onSwitch} label="I've written down the information above." />
+      </p>
+      <rb.Button
+        variant="dark"
+        type="submit"
+        disabled={!userConfirmed}
+        onClick={() => userConfirmed && walletConfirmed()}
+      >
+        Fund Wallet
```

Comment:
```suggestion
        Fund wallet
```
In Figma (if I am on the correct version) "wallet" is not capitalized.
I do not think it is relevant.. it was the only real difference I saw to an otherwise pixel-perfect implementation :muscle: